dds_cli/project_status.py

```diff
@@ -148,6 +149,108 @@ def update_status(self, new_status, deadline=None, is_aborted=False, no_mail=Fal
 
         dds_cli.utils.console.print(f"Project {response_json.get('message')}")
 
+    def extend_deadline(self, new_deadline=None):
+        """Extend the project deadline."""
+        # Define initial parameters
+        extra_params = {"send_email": False}
+
+        # Fetch project status and default deadline
+        response_json, _ = dds_cli.utils.perform_request(
+            endpoint=DDSEndpoint.UPDATE_PROJ_STATUS,
+            headers=self.token,
+            method="patch",
+            params={"project": self.project},
+            json=extra_params,
+        )
+
+        # Structure of the response:
+        #   {
+        #   'default_unit_days': 30,
+        #   'project_info': {
+        #       'Created by': 'First Unit User',
+        #       'Description': 'This is a test project',
+        #       'Last updated': 'Wed, 18 Oct 2023 08:40:43 GMT',
+        #       'PI': 'support@example.com',
+        #       'Project ID': 'project_1',
+        #       'Size': 0,
+        #       'Status': 'Available',
+        #       'Title': 'First Project'
+        #       },
+        #   'project_status': {
+        #       'current_deadline': 'Sat, 04 Nov 2023 23:59:59 GMT',
+        #       'current_status': 'Available'},
+        #   'warning': 'Operation must be confirmed before proceding.'
+        #   }
+
+        # Extract default unit days and current deadline
+        default_unit_days = response_json.get("default_unit_days")
+        current_deadline = response_json.get("project_status").get("current_deadline")
+        project_id = response_json.get("project_info").get("Project ID")
+        # print information about the project status and table with the project info
+        print_info = (
+            f"\nCurrent deadline: [b][green]{current_deadline}[/green][/b]\n"
+            f"Default deadline extension: [b][green]{default_unit_days}[/green][/b] days\n"
+        )
+        table = self.generate_project_table(project_info=response_json.get("project_info"))
+        dds_cli.utils.console.print(table)
+        dds_cli.utils.console.print(print_info)
+
+        # If it wasnt provided during the command click, ask the user for the new deadline
+        if not new_deadline:
+            # Question number of days to extend the deadline
+            prompt_question = (
+                "How many days would you like to extend the project deadline with? "
+                f"Leave empty in order to choose the default ([b][green]{default_unit_days}[/green][/b])."
+            )
+            new_deadline = rich.prompt.Prompt.ask(prompt_question)
+            if not new_deadline:
+                # Set new_deadline to default
+                new_deadline = default_unit_days
```

Comment:
You can use the `default` option provided by `Prompt.ask` instead of having this extra check
```suggestion
            new_deadline = rich.prompt.Prompt.ask(prompt_question, default=default_unit_days)
```